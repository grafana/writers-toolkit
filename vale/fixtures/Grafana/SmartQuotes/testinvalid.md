Don't use:

- ‘ (left single quotation mark)
- ’ (right single quotation mark)
- ʼ (modifier letter apostrophe)
- “ (left double quotation mark)
- ” (right double quotation mark)
