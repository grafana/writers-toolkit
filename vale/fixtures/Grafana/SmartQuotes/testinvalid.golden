testinvalid.md:3:3:Grafana.SmartQuotes:Avoid smart quotes in the source file, especially in code blocks. In some contexts, Unicode characters aren't supported and break configurations. The website renders paired quotes using smart quotes in paragraphs. 
testinvalid.md:4:3:Grafana.SmartQuotes:Avoid smart quotes in the source file, especially in code blocks. In some contexts, Unicode characters aren't supported and break configurations. The website renders paired quotes using smart quotes in paragraphs. 
testinvalid.md:5:3:Grafana.SmartQuotes:Avoid smart quotes in the source file, especially in code blocks. In some contexts, Unicode characters aren't supported and break configurations. The website renders paired quotes using smart quotes in paragraphs. 
testinvalid.md:6:3:Grafana.SmartQuotes:Avoid smart quotes in the source file, especially in code blocks. In some contexts, Unicode characters aren't supported and break configurations. The website renders paired quotes using smart quotes in paragraphs. 
testinvalid.md:7:3:Grafana.SmartQuotes:Avoid smart quotes in the source file, especially in code blocks. In some contexts, Unicode characters aren't supported and break configurations. The website renders paired quotes using smart quotes in paragraphs. 
